Mindee Invoice.ipynb

```diff
@@ -0,0 +1,417 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(parsed_data.http_response)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "minute-charger",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "26.04\n",
+      "4.95\n",
+      "30.99\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(parsed_data.invoice.total_excl.value)\n",
+    "print(parsed_data.invoice.total_tax.value)\n",
+    "print(parsed_data.invoice.total_incl.value)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "complicated-asian",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " tax amount: 4.95 \n",
+      " tax_code: None \n",
+      " tax_rate: 19.0 \n"
+     ]
+    }
+   ],
+   "source": [
+    "# To get the list of taxes\n",
+    "taxes = parsed_data.invoice.taxes\n",
+    "\n",
+    "# Loop on each Tax field\n",
+    "for tax in taxes:\n",
+    "   # To get the tax amount\n",
+    "   tax_amount = tax.value\n",
+    "   \n",
+    "   # To get the tax code for from a tax object\n",
+    "   tax_code = tax.code\n",
+    "  \n",
+    "   # To get the tax rate\n",
+    "   tax_rate = tax.rate\n",
+    "   print((\" tax amount: {tax_amount} \\n tax_code: {tax_code} \\n tax_rate: {tax_rate} \").format(tax_amount=tax_amount, tax_code=tax_code, tax_rate=tax_rate ))"
```

Comment:
Since there might be more items, I'd suggest:
```python
for tax_item in taxes:
    print(f"{tax_item.code} - rate {tax_item.rate / 100:.2%} - amount {tax_item.value:.2f}")
```

Reply:
added

Reply:
It was a replacement suggestion :) 
You can remove the other lines of the block

Reply:
done